nemo/core/classes/common.py

```diff
@@ -486,7 +486,7 @@ def from_config_dict(cls, config: 'DictConfig', trainer: Optional['Trainer'] = N
             # target class resolution was unsuccessful, fall back to current `cls`
             if instance is None:
                 if imported_cls_tb is not None:
-                    logging.debug(
+                    logging.info(
```

Comment:
This should remain debug, it prints a lot of information otherwise which just confused users, even though model is properly instantiated.

Reply:
That was for debug - I will revert that.

Reply:
It is still in debug mode.